components/kyma-environment-broker/common/hyperscaler/account_pool.go

```diff
@@ -109,7 +109,7 @@ func (p *secretBindingsAccountPool) IsSecretBindingUsed(hyperscalerType Type, te
 }
 
 func (p *secretBindingsAccountPool) CredentialsSecretBinding(hyperscalerType Type, tenantName string) (*v1beta1.SecretBinding, error) {
-	labelSelector := fmt.Sprintf("tenantName=%s,hyperscalerType=%s", tenantName, hyperscalerType)
+	labelSelector := fmt.Sprintf("tenantName=%s, hyperscalerType=%s, !dirty", tenantName, hyperscalerType)
```

Comment:
After additional review I think we should make in both cases (L112 and L 126) same label selector. `shared!=true, tenantName=%s, hyperscalerType=%s, !dirty` . I'm not sure if we can add `internal=true` label to selector query.

Reply:
Resolved!